wiki/Configuration:-Input.md

```diff
@@ -124,6 +127,9 @@ A few settings are common between `touchpad`, `mouse` and `trackpoint`:
 - `natural-scroll`: if set, inverts the scrolling direction.
 - `accel-speed`: pointer acceleration speed, valid values are from `-1.0` to `1.0` where the default is `0.0`.
 - `accel-profile`: can be `adaptive` (the default) or `flat` (disables pointer acceleration).
+- `scroll-method`: when to generate scroll events instead of pointer motion
+  events, can be `none`, `two-finger`, `edge`, or `on-button-down`. The default
+  and supported methods vary depending on the device type.
```

Comment:
I wrap the md files in this repo by sentence.

```suggestion
- `scroll-method`: when to generate scroll events instead of pointer motion events, can be `none`, `two-finger`, `edge`, or `on-button-down`.
  The default and supported methods vary depending on the device type.
```